backend/tests/test_auth_security.py

```
@@ -7,34 +7,58 @@
 from datetime import datetime, timedelta
 from flask import current_app
 from models import User
+from auth import JWTManager
+
+
+def extract_token_from_cookie(response, token_name='access_token'):
```

Ruff (0.14.1)

Possible hardcoded password assigned to function default: "token_name"

(S107)